app/__init__.py

```diff
@@ -109,4 +109,14 @@ def show_state_locations():
     state = request.args.get('state')
 
     locations = get_testing_locations(state, API_BASE_URL)
-    return render_template('location.html', locations=locations)
\ No newline at end of file
+    return render_template('location.html', locations=locations)
```

Comment:
How is this change related to this PR ?

Reply:
I'm having trouble understanding the issue with this code. I am adding new code underneath `return render_template('location.html', locations=locations)`. Maybe that is why this line of code keeps showing up? I am not adding anything new to the mentioned line.

Reply:
I guess this issue is still under review [https://github.com/rtfpessoa/diff2html/issues/157](https://github.com/rtfpessoa/diff2html/issues/157) We shall get back to it later.